client/src/components/ProjectSearchFilter.js

```diff
@@ -89,6 +91,26 @@ export const ProjectSearchFilter = ({
 
   return (
     <Box overflow="auto">
+      {filterOptions.models.map((f, i, arr) => (
+        <Box
+          key={f}
+          border={!isLastIndex(i, arr) ? false : { side: 'bottom' }}
+          pad={{
+            top: `${i === 0 ? 'medium' : 'none'}`,
+            bottom: `${!isLastIndex(i, arr) ? 'none' : 'medium'}`
+          }}
+          height={{ min: 'auto' }}
+        >
+          <Text weight="bold">{filterNames[f]}</Text>
```

Comment:
The models group doesn't have headings in the designs, just a couple of checkboxes stacked at the top.